### PR TITLE
Nofuture

### PR DESCRIPTION
**Pull request by wwlytton**

### CODE REVIEW

File: netpyne/network/conn.py [hunk 1]
```diff
@@ -26,12 +20,12 @@
 from numbers import Number
 from numpy import array, sin, cos, tan, exp, sqrt, mean, inf, dstack, unravel_index, argsort, zeros, ceil, copy 
 
-
 # -----------------------------------------------------------------------------
 # Connect Cells
 # -----------------------------------------------------------------------------
 def connectCells (self):
     from .. import sim
+    h = sim.h; pc = h.ParallelContext[0]; idhost, nhost, cvode = pc.id(), pc.nhost(), h.CVode()
```
Comment (joewgraham) [inline netpyne/network/conn.py:28]: What does this addition do?  Is it necessary for some reason?

File: netpyne/network/conn.py [hunk 2]
```diff
@@ -52,6 +46,7 @@ def connectCells (self):
 
     gapJunctions = False  # assume no gap junctions by default
 
+    print("AAAAAAAAAAAAAAAA %d %d"%(idhost,nhost),[x for x in self.params.connParams.items()])
```
Comment (joewgraham) [inline netpyne/network/conn.py:49]: Maybe get rid of this...

File: netpyne/network/conn.py [hunk 3]
```diff
@@ -1,16 +1,10 @@
-
 """
 network/conn.py 
 
 Network class methods to create connections 
 
 Contributors: salvadordura@gmail.com
 """
-from __future__ import print_function
```
Comment (joewgraham) [inline netpyne/network/conn.py:9]: These all seem to be boilerplate to ensure Python 2 <--> 3 compatibility.  I suggest we leave them in unless there is a reproducible problem with them...

**Comment by wwlytton** [comment]
print "AAAA...."  -- whoops sorry

File: netpyne/network/conn.py (hunk 3 above)
Comment (wwlytton) [reply netpyne/network/conn.py:9]: i'm going to now try with new installs -- current netpyne, current ubuntu 20.04, and wheel-installed NEURON

Comment (joewgraham) [reply netpyne/network/conn.py:9]: Sounds good. Let us know what happens.

**Comment by salvadord** [comment]
@joewgraham - looks like here still waiting for @wwlytton to clarify: 
1) why this is needed? `h = sim.h; pc = h.ParallelContext[0]; idhost, nhost, cvode = pc.id(), pc.nhost(), h.CVode()`

2) if any reason to remove the `from __future__` imports

**Comment by wwlytton** [comment]
not seeing this error anymore after update to ubuntu 20.04 ; python Python 3.8.2

**Pull request closed without merging**